zebra-node-services/src/scan_service/request.rs

```diff
@@ -22,5 +27,27 @@ pub enum Request {
     SubscribeResults(Vec<()>),
 
     /// TODO: Accept `KeyHash`es and return transaction ids
-    ClearResults(Vec<()>),
+    ClearResults(Vec<String>),
```

Comment:
We're missing docs here.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/8219/commits/70e176e748f415cb6d977b7e09c93fd802a5b6f3.